tests/model/test_fully_connected.py

```diff
@@ -22,10 +22,13 @@ def fully_connecteds(
     st.SearchStrategy[Tuple[FullyConnected, Dict]],
 ]:
     """Returns a SearchStrategy for FullyConnected."""
+    eps = 1e-8
     kwargs = {}
     kwargs["in_features"] = draw(st.integers(1, 32))
     kwargs["out_features"] = draw(st.integers(1, 32))
     kwargs["num_hidden_layers"] = draw(st.integers(0, 8))
+    kwargs["dropout"] = None
```

Comment:
Yeah that's much better. 

The `eps` was a holdover from a time when in the proto I was using:

`float dropout = 4;`
instead of:
`google.protobuf.FloatValue dropout = 4;`
And hence the protobuf compilation added some `eps` to the dropout value.